webapp/niapp.yaml

```diff
@@ -0,0 +1,416 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: SystemLink WebApp Service
+basePath: /niapp/v1
```

Comment:
@spanglerco will putting V1 in the basePath cause any issues with NI webserver on SystemLink Server?

If not, I actually really like putting the version in the basePath, especially given the precedent of using a separate swagger doc for different API versions. It seems to remove duplication a lot. We should just double-check that it will work correctly with NI web server, especially given that this implies a separate web service for each API version (basePath corresponds to web service).

Reply:
No, it won't cause any problems. The only thing that we don't support is mixing variable and static text in a single segment (e.g. /v{version}).